RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHGC_cfi.py

```diff
@@ -48,7 +48,18 @@
     algoName = cms.string("PFClusterFromHGCalMultiCluster"),
     thresholdsByDetector = cms.VPSet(
     ),
-    clusterSrc = cms.InputTag("hgcalMultiClusters")
+    
+    #clusterSrc = cms.InputTag("hgcalMultiClusters"),
+    #clusterSrc = cms.InputTag("multiClustersFromTrackstersEM"),
+    #clusterSrc = cms.InputTag("ticlMultiClustersFromTrackstersEM"),
+    clusterSrc = cms.InputTag("ticlMultiClustersFromTrackstersMerge"),
+    
+    tracksterSrc = cms.InputTag("ticlTrackstersMerge"),
+    
+    filterByTracksterPID = cms.bool(False),
+    pid_threshold = cms.double(0.8),
+    filter_on_categories = cms.vint32([0, 1]),
```

Comment:
@jpata I just realized that the type-free syntax does not work for here as these parameters are not already defined.
Doing so will require adding their descriptions to the base producer, if I am not mistaken.
However, I feel it would be better and easier to not do so for now, so that the PFClusterFromHGCalMultiCluster module can be edited and tested easily w/o disturbing the base producer.
I have updated the PR removing the type-free syntax from this portion, however, please let me know if you strongly feel otherwise.

Reply:
@SohamBhattacharya @jpata : yes, here we are defining a PSet from scratch, to be included later on in actual configs: therefore the type must be defined.

Reply:
@perrotta Just to make sure I understand correctly, you mean this okay, or I need to define the parameters in a fillDescriptions function?
The latter does not seem to have been done for the other PSets defined here, so I was a bit confused.
If it's the latter, can you please point me to the .cc file where it would be most appropriate to add the descriptions?

Reply:
@SohamBhattacharya the intention of my message was to confirm what you also wrote, that you cannot remove the definition of the types from these configs. Having them defined with a fillDescription method somewhere is welcome, but I did not suggest it, and I did not even investigate whether it is easily feasible. My suggestion is to let things here as they are now (just cleaning a bit the configs from unneeded empty lines, as I suggested in another comment, would also be welcome, for an easier reading)

Reply:
Right, you can't remove them here - I didn't see the lines above that you are defining a new PSet rather than overriding (as was the case elsewhere). Please implement Andrea's suggestion of cleaning the whitespace, though!

Reply:
@perrotta @jpata thanks for the clarifications. I have implemented the changes and updated the PR.